docs/RFCS/typing.md

```diff
@@ -725,23 +755,24 @@ These situations may or may not also desire a given type for all subexpressions.
 examples of this type of situation are in CASE statements (both for the condition set and the
 value set) and in COALESCE statements. Because this is a common need for a number of statement
 types, the typing resolution of this situation should be specified. Here we present a list of
-rules to be applied to a given list of untyped expressions and an optional desired type.
+rules to be applied to a given list of untyped expressions and an optionally-specified desired type.
```

Comment:
what does "optionally-specified" mean here? (I don't think you've defined this so far)